interface/patient_file/pos_checkout.php

```diff
@@ -1096,8 +1096,7 @@ function computeTotals() {
                         <div class="form-group">
                             <div class="col-sm-12 text-left position-override">
                                 <div class="btn-group btn-group-pinch" role="group">
-                                    <!--<input type='submit' class="btn btn-default btn-save"  name='form_save' id='form_save' value='<?php echo xla('Save'); ?>' />-->
-                                    <button type='submit' class="btn btn-default btn-save"  name='form_save' id='form_save' ><?php echo xla('Save'); ?></button>
+                                    <button type='submit' class="btn btn-default btn-save"  name='form_save' id='form_save' value='<?php echo xla('Save'); ?>'><?php echo xlt('Save');?></button>
```

Comment:
It's never a good idea to translate values especially submits. Many times the script being posted to will check the value of the button to determine if a save(or other action) is needed. This comparison value will be hardcoded in the language of the author(mostly english) so sending a value in say spanish, will fail.
Hope this is clear.

Reply:
so just use `text()`?

Reply:
escaping really not needed here. just make it value="1" in the button element.
However i'm concerned why this is occurring. Did Zbig change this during a restyle by removing the value attr or did the POST conditional get changed? The POST conditional may have been isset($_POST['form_save']) which would work so this concerns me that this self imposed imposed issue may be elsewhere in codebase.
I'm removing change requested and thanks.

Reply:
Yes, zbig restyled but there was no value attribute, https://github.com/openemr/openemr/commit/41f84b65823d5475cdcdc3191a1bd6ae9b8ad027#diff-155a720336fe1b77a1cdb0b27cc47622

Reply:
Make it:
`<button type='submit' class="btn btn-default btn-save"  name='form_save' id='form_save' value='save'><?php echo xlt('Save');?></button>`


> Yes, zbig restyled but there was no value attribute,

Okay so how did it ever work?

Reply:
:tophat: :rabbit:

Reply:
rabbit+hat. do you mean pull a rabbit out of our hat? or are you having rabbit for dinner tonight? :)

Reply:
:stew:

Reply:
it works magically :)